### DISCUSSION TITLE
How to install APK's

### DISCUSSION THREAD

**Discussion by L3kSUS** [MustardChef/WSABuilds#393] [Q&A] [answered]
I wanna install APK on my WSA. How can i do this?

**Comment by MustardChef** [comment, marked as answer]
> I wanna install APK on my WSA. How can i do this?

[Follow this guide.](https://github.com/MustardChef/WSABuilds/blob/master/Documentation/Usage%20Guides/Sideloading%20Guides/Sideloading.md)
